templates/cli/README.md.twig

```diff
@@ -75,6 +75,16 @@ $ {{ language.params.executableName }} users create --email="hello@{{ spec.title
 $ {{ language.params.executableName }} users list 
 ```
 
+To create a Document you can use the following command 
+```sh
+$ {{ language.params.executableName }} database createDocument --collectionId="YOUR COLLECTION ID" --data='A VALID JSON STRING' --read=role:member --read="*" --write=role:guest
+```
+
+### Some Gotchas
+- `data` expects the JSON string to be escaped.
+- If using the wildcard (`*`) read or write permissions , make sure that it is properly escaped using a `\` or by enclosing it in `"*"`.
+- Some arguments like `read` and `write` permissions are expected to be arrays. In the {{ spec.title }} CLI arrays are passed by simply repeating the argument as seen in the `createDocument` example above.
```

Comment:
In the {{ spec.title }} CLI -> `In the {{ spec.title }} CLI, `

Reply:
Done :D